src/ServiceBus.AttachmentPlugin/ServiceBus.AttachmentPlugin.csproj

```diff
@@ -5,9 +5,12 @@
     <Version>6.0.0-preview01</Version>
     <Authors>Sean Feldman</Authors>
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <LangVersion>latest</LangVersion>
+    <Nullable>enable</Nullable>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <PackageTags>Azure;Service Bus;ServiceBus;.NET;AMQP;IoT;Queue;Topic;Attachment;Plugin</PackageTags>
-    <PackageIconUrl>https://raw.githubusercontent.com/SeanFeldman/ServiceBus.AttachmentPlugin/master/images/project-icon.png</PackageIconUrl>
-    <PackageLicense>https://raw.githubusercontent.com/SeanFeldman/ServiceBus.AttachmentPlugin/master/LICENSE</PackageLicense>
+    <PackageIcon>project-icon.png</PackageIcon>
+    <PackageLicenseFile Condition="'$(PackageLicenseFile)' == ''">LICENSE.md</PackageLicenseFile>
```

Comment:
move to a package license expression
eg
```
<PackageLicenseExpression>MIT</PackageLicenseExpression>
```

Reply:
What's the benefit over including the file?

Reply:
nuget can parse and use the expression. as well as others doing analysis on packages

Reply:
What about the `LICENSE.md` file? Should it still remain or it's not needed if the expression is in place?

Reply:
yep u still need a license md in the source since github hasn’t adopted the license expression standard

Reply:
yep u still need a license md in the source since github hasn’t adopted the license expression standard